webpack.base.config.js

```diff
@@ -189,7 +189,8 @@ module.exports = (env) => {
               options: {
                 implementation: require("sass"),
                 sassOptions: {
-                  indentedSyntax: true
+                  indentedSyntax: true,
+                  additionalData: `$product: ${product};`,
```

Comment:
Why not define `$is-codecombat` already here so we can skip these 
https://github.com/codecombat/codecombat/pull/7014/files#diff-ffdec86f389b986608e7acf6442922c659476fae0afddc7e810204f93d0c8673R388
?

Reply:
emmm the `additionalData` seems only can pass string in instead of boolean so ... anyway we need a check later 😂

Reply:
can't you say something like this?
```
additionalData: `$is-codecombat: $product == '${product}';`
```